Thesis.tex

```diff
@@ -60,6 +60,17 @@ \section{Copyright}
 
 \section{Patents}
 \label{sec:Patents}
+According to Wikipedia, a patent is "a set of exclusive rights granted by a sovereign state to an inventor or assignee for a limited period of time in exchange for detailed public disclosure of an invention. [...]  Patents are a form of intellectual property." (citation needed https://en.wikipedia.org/wiki/Patent Abruf am 27.3.17)
+
+Patents are thus used to encourage disclosure of new invention - thus making it available for others to spur further developments while at the same time granting the patent-holder privileges to cash in on the invention. The idea behind it can e.g. be found in the Copyright and Patent clause, Article I, Section 8, Clause 8 of the United States Constitution: "The Congress shall have Power [...] To promote the Progress of Science and useful Arts, by securing for limited Times to Authors and Inventors the exclusive Right to their respective Writings and Discoveries [...]" (citation needed https://www.senate.gov/civics/constitution_item/constitution.htm#a1_sec8 Constitution of the United States).
+
+It is important to note that the patent holder was never intended to be the sole profiteer of the patent deal, instead creation of patents was intended to be a deal "quid pro quo" where on one side of the balance, the patent holder got a limited monopoly while the public on the other side of the balance got the invention in a way that would make it usable and, after the monopoly has ended, reproducable. This was established in various court opinions and orders, even as early as 1858, when Judge Daniels wrote in Kendall v. Winsor (62 .S. 322): 
```

Comment:
I would change the deal to _... was intended to be a "quid pro quo" deal, where_, as the "quid pro quo" makes the "deal" more specific and is acting as an adjective